### DISCUSSION TITLE
Masonry layouts tailwind

### DISCUSSION THREAD

**Discussion by elmeftouhi** [tailwindlabs/tailwindcss#2388] [Ideas]
Hey Guys, 
Is there any one how has already implemented tailwind to create a Masonry layouts.
Thank you!!

**Comment by mohit2404** [comment]
you can use columns-1 md:column-2 to have a masonry layout and the contents can have different heights

**Comment by rozsazoltan** [comment]
https://play.tailwindcss.com/csuN6n5xFA
